docs/man/git-lfs-migrate.1.ronn

```diff
@@ -10,6 +10,9 @@ git-lfs-migrate(1) - Migrate history to or from git-lfs
 * `info`
     Show information about repository size.
 
+* `import`
```

Comment:
❇️ Can we put this above the `info` mode, since `import` sorts ahead of `info` lexicographically?

Reply:
I was thinking it'd be better to sort _logically_. `info` is the first command you run. 

I imagine we'll add an `export` command at some point. What looks better?

* lexicographically
  * export
  * import
  * info
* technologic
  * info
  * import
  * export

Reply:
Ah, great call! I really dig the technological ordering :)